static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -59,7 +59,7 @@ export function InviteMissingMembersModal({
 
   const api = useApi();
 
-  if (!memberInvites || !organization.access.includes('org:write')) {
+  if (!memberInvites.length || !organization.access.includes('org:write')) {
```

Comment:
js has weird ways of treating objects as boolean. Can you make this more clear like
`memberInvites && memberInvites.length > 0`?